pkg/rtc/room.go

```diff
@@ -774,7 +774,7 @@ func (r *Room) CloseIfEmpty() {
 	if r.FirstJoinedAt() > 0 && r.LastLeftAt() > 0 {
 		elapsed = time.Now().Unix() - r.LastLeftAt()
 		// need to give time in case participant is reconnecting
-		timeout = RoomDepartureGrace
```

Comment:
Can we delete this constant? Guess it is used in some test. Wondering if the test can be changed to not use that constant. Would be good to get rid of constants that might be confusing.